screenpipe-server/src/core.rs

```diff
@@ -237,6 +242,19 @@ async fn record_video(
                         } else {
                             &window_result.text
                         };
+
+                        let _ = realtime_vision_sender.send(RealtimeVisionEvent::WindowOcr(
+                            WindowOcr {
+                                // image: frame.image.clone(),
```

Comment:
whats the problem with image? too much CPU? too much RAM?

in /search we make the frame optional because it adds lot of delay 

we could make streaming frames optional too in SSE/streaming? default to false

Reply:
RAM usage. I'll make it optional